### DISCUSSION TITLE
Stellarium licensing for livestream

### DISCUSSION THREAD

**Discussion by kulp95** [Stellarium/stellarium#2276] [Q&A]
Hi there, I'm a STEM educator and wondering if it is okay to use the Stellarium software on an educational livestreaming channel. I would be screen-sharing my desktop running Stellarium, using it to supplement astronomy activities. I read on your FAQ that some of the textures may be copyrighted(?) but I'm not sure yet if I will be using them. Is this okay, and if so, how should I credit you all properly?

**Comment by gzotti** [comment]
Yes, probably hundred of astro clubs worldwide meanwhile use it also for online channels, no problem. All we ask for is that you say somewhere "simulation with Stellarium", either in your narration and/or in the final credits.

**Reply by kulp95** [reply]
Thanks, I will make sure to do that. Is it okay even if the streams are monetized?

**Reply by gzotti** [reply]
I think if the story that you are telling is yours, this is OK. Like you won't ask the typewriter manufacturer if it's OK to author a book on the machine.  But then proper crediting should be seen mandatory. @alex-w ?

**Reply by kulp95** [reply]
Yes, thank you for the info! Could I give credit verbally or would you prefer I have it written in the description of the channel (or both)?

**Reply by gzotti** [reply]
Yes, why not both? ;-)
